### PR TITLE
Update Leap

### PR DESCRIPTION
**Pull request by iHiD**
This:
- Rewords the copy to make the instructions a little less algorithmic in nature
- Brings the copy in line with other recent updates
- Splits the copy into two introduction/instructions

### CODE REVIEW

File: exercises/leap/introduction.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Introduction
+
+A leap year (in the Gregorian calendar) occurs in every year that is evenly divisible by 4, except if the year is evenly divisible by 100, unless the year is also evenly divisible by 400.
```
Comment (IsaacG) [inline exercises/leap/introduction.md:3]: This sentence is a bit tricky to parse with the except-unless clause. Do those both apply to the base rule? Does the unless apply to except? Does except and unless mean something different?
I'm wondering if it makes sense to split this into multiple sentences and make the exceptions a bit more explicit.

Comment (vaeng) [reply exercises/leap/introduction.md:3]: Maybe like:

> A leap year (in the Gregorian calendar) occurs for every year evenly divisible by 4, but there are two exceptions.
If the year is evenly divisible by 100 it is not considered a leap year, unless it is also evenly divisible by 400.

Comment (iHiD) [reply exercises/leap/introduction.md:3]: How about:

```suggestion
A leap year (in the Gregorian calendar) occurs:
- In every year that is evenly divisible by 4
- Unless the year is evenly divisible by 100, in which case it's only a leap year if the year is also evenly divisible by 400.
```

File: exercises/leap/instructions.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Instructions
+
+Your task is to determine if any given year is a leap year.
```
Comment (vaeng) [inline exercises/leap/instructions.md:3]: I like that this is now more on track with the "modern" concept exercise style files

**Review by vaeng — APPROVED** [review]

File: exercises/leap/introduction.md (hunk 1 above)
Comment (IsaacG) [reply exercises/leap/introduction.md:3]: That looks good to me!

**Review by BethanyG — APPROVED** [review]

**Review by IsaacG — APPROVED** [review]

**Review by ErikSchierboom — APPROVED** [review]

**Review by kotp — APPROVED** [review]

**Comment by iHiD** [comment]
Thanks everyone! Updating your tracks to the latest version would be greatly appreciated :)

**Pull request merged**